docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+For example - You add a new Integration to your SaaS platform. Enter 30 here to limit the access token validity to ``30`` minutes. If you haven’t completed the authentication in that period of time, you must revalidate.
+
+* **Refresh token lifetime** - When using OAuth 2.0, the lifetime of the refresh token used to request a new access token once expired. Once the refresh token expires, you must reauthorize. The default is ``14`` days.
+
 Asset settings
 **************
 
+.. image:: images/assets-settings.png
+  :width: 600
+  :alt: Screenshot showing Assets Settings Configuration in Mautic
+
+* **Path to the Asset directory** - Set the absolute path to where Assets is uploaded. In order to prevent the public from accessing Assets, use a directory outside of the public web root.
+
+* **Maximum size (MB)** - Set the maximum size of uploaded Assets in MB (Megabytes).
+
+* **Allowed file extensions** - Extensions of files separated by commas. You can only upload files with the specified file extensions.
+
 Campaign settings
 *****************
 
+.. image:: images/campaign-settings.png
+  :width: 600
+  :alt: Screenshot showing Campaign Settings Configuration in Mautic
+
+* **Wait time before retrying a failed action** - If for any reason a Campaign action doesn't execute, this is the length of time Mautic waits before trying again.
+
+* **Use date range for all views** - When viewing a Campaign, the actions, conditions, decisions, and Contacts tabs is based on the date range provided.
+
+* **Use summary statistics** - Improves performance when viewing a Campaign with thousands of events per day by using summarized data.
+
 Email settings
 **************
 
 Mail send settings
 ==================
 
+.. image:: images/mail-send-settings.png
+  :width: 600
+  :alt: Screenshot showing Mail Send Settings Configuration in Mautic
+
+* **Name to send mail as** - The default name Emails come from. This is typically something like ``{YourCompany}`` Marketing Team or ``{YourCompany}``.
+
+* **Email address to send mail from** - The Email address for the name you’re sending mail from. The address displays in the From: field when your Contacts receive your Emails.
+
+.. note::
+
+   Ensure that you configure your sender domain, DKIM, bounce, and click tracking domains. For more information, see Email domains document.
+
+* **Reply to address** -  To have Contacts reply to a different address than the address defined in Email address to send mail from, add the desired address to receive replies. This is the default address where messages are sent whenever Contacts reply to messages sent from Mautic. If this field is blank, the address specified in **Email address to send mail** from is used. This is useful if your configured sender domain, which you use in the from address, contains a subdomain that doesn't have MX records or is otherwise an address that can't receive Emails.
```

Comment:
```suggestion
* **Reply to address** -  To have Contacts reply to a different address than the specified From address, add the desired address here. This is the default reply-to address where messages are sent from Mautic unless it is overridden in an individual email. If this field is blank, the address specified in **Email address to send mail** from is used. The reply-to setting is useful if your configured sender domain - which you use in the from address - contains a subdomain that doesn't have MX records or is otherwise an address that can't receive Emails.
```